src/dialog_style_manager.cpp

```diff
@@ -704,18 +757,27 @@ void DialogStyleManager::OnCurrentImport() {
 	int res = GetSelectedChoices(this, selections, _("Choose styles to import:"), _("Import Styles"), to_wx(styles));
 	if (res == -1 || selections.empty()) return;
 	bool modified = false;
+	bool overwriteAll = false;
+	int n = selections.GetCount();
 
 	// Loop through selection
-	for (auto const& sel : selections) {
+	for (int i =0; i < n; ++i) {
+		auto const& sel = selections.at(i);
+		auto const styleName =  styles[sel];
+
 		// Check if there is already a style with that name
 		if (AssStyle *existing = c->ass->GetStyle(styles[sel])) {
-			int answer = wxMessageBox(
-				fmt_tl("There is already a style with the name \"%s\" in the current script. Overwrite?", styles[sel]),
-				_("Style name collision"),
-				wxYES_NO);
-			if (answer == wxYES) {
+
+			bool overwriteThis = false;
+			if(!overwriteAll){
+				StyleOverrideFlag overwriteFlag = DisplayStyleOverrideDialog(n-i, styleName, "script");
+			overwriteThis = overwriteFlag == StyleOverrideFlag::Yes;
```

Comment:
This line is mis-indented.

Reply:
👍🏼 
This repo has no Clang-format or similar, so no automatic formatting is possible, so I did it by hand, I missed that 😓